config/api.yml

```diff
@@ -4107,6 +4114,11 @@
       - :name: read
         :identifier:
         - vm_show
+    :compliances_subcollection_actions:
+      :get:
+      - :name: read
+        :identifier:
+        - vm_show
```

Comment:
compliances are technically on a bunch of different resources.  @abellotti Is there a way to do an "OR" of identifiers?  Not sure what the right identifiers are, but I usually go with whatever the UI does... @himdel ?

Reply:
we can specify multiple identifiers (essentially the OR) via :identifiers instead of :identifier

Reply:
Note, I'm more thinking of if this is a subcollection of more than just vms.

Reply:
There's a bunch of `*_check_compliance` features that the UI uses for the "Policy / Check Compliance of the last known configuration"  toolbar button.

So, I guess `instance_check_compliance`, `image_check_compliance`, `vm_check_compliance` and `miq_template_check_compliance` here?

Reply:
thanks, I'll get to `miq_template_check_compliance` in a follow-up